src/audio/tdfb/tdfb.c

```diff
@@ -116,14 +122,39 @@ static void tdfb_free_delaylines(struct tdfb_comp_data *cd)
 		fir[i].delay = NULL;
 }
 
+static void *tdfb_filter_seek(struct sof_tdfb_config *config, int num_filters)
+{
+	struct sof_fir_coef_data *coef_data;
+	int i;
+	uint8_t *coefp = (uint8_t *)ASSUME_ALIGNED(&config->data[0], 2);
```

Comment:
why aligned to 2? i feel like this probably should have a comment why its not 4

Reply:
Yep, comment needed. The struct is entirely int16_t, so aligned to 2 is sufficient. In HiFi3 the aligned int16x4 load worked efficiently so it did not need to become aligned to 8. It may need to be revisited in future with other processors (x86?), and move this to FIR module side.

Reply:
Comment added.